framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -207,6 +207,11 @@
     // java.lang.String)
     "requirePrefixInWarningSuppressions",
 
+    // Ignore annotations in bytecode that have invalid annotation locations.
```

Comment:
A note in the manual should also be added.

Reply:
Done.